lib/reporters/mini.js

```diff
@@ -51,15 +85,18 @@ MiniReporter.prototype.test = function (test) {
 		this.passCount++;
 	}
 
-	status += '  ' + title;
+	var status = '  ' + title;
 	status += '\n\n';
 
 	if (this.passCount > 0) {
-		status += '  ' + colors.pass(this.passCount, 'passed');
+		status += '   ' + colors.pass(this.passCount, 'passed');
 	}
 
 	if (this.failCount > 0) {
-		status += '  ' + colors.error(this.failCount, 'failed');
+		if (this.passCount > 0) {
+			status += '\n';
+		}
```

Comment:
Good call on `todo`. I implemented this before rebasing, so yeah - that almost certainly needs a fix.

Are skipped tests shown yet? I thought there was still a pending PR for that.